our-work/software/nlp-syllabi.yml

```diff
@@ -0,0 +1,18 @@
+title: Natural Language Processing of Brown Course Syllabi 
+description: "Web dahsboard that allows Brown University Library staff to interactively use natural language processing for investigating how race and racial issues are discussed in courses at Brown." 
+languages: 
+   - python
+   - R 
+tags:
+   - web
+   - machine-learning 
+groups:
+   - brown-university-library
+people:
+   - name: Jordan Lawson
+     github_user: j-stat
+links:
+   - category: code
+     url: https://github.com/compbiocore/NLP_Syllabi_Project 
```

Comment:
This link leads to a 404. Is it private? Can it be public? If not, let's just delete these lines

Reply:
This might need to remain private, let me double-check with Ashok and get back to this. Thanks!

Reply:
Checked, we're good and this is public now!